app/views/idv/in_person/address.html.erb

```diff
@@ -96,22 +68,6 @@
         ) %>
   </div>
 
-  <% unless capture_secondary_id_enabled %>
-    <%= render ValidatedFieldComponent.new(
-          as: :radio_buttons,
-          checked: pii[:same_address_as_id],
-          collection: [
-            [t('in_person_proofing.form.address.same_address_choice_yes'), true],
-            [t('in_person_proofing.form.address.same_address_choice_no'), false],
-          ],
-          form: f,
-          label: t('in_person_proofing.form.address.same_address'),
-          name: :same_address_as_id,
-          required: true,
-          wrapper: :uswds_radio_buttons,
-        ) %>
-  <% end %>
-
```

Comment:
I have not spun this up yet so sorry if I am confusing the logic. (I will delete or edit this comment once I have tested this out.)

Should we just delete the <% unless capture_secondary_id_enabled %> & <% end %> pieces and display the inside render?  (Same for the other 2 unless blocks of code in this html file.) I may have goofed up logic- or you may be getting to this.  When I start up sever I will test this out.